docs/src/increase-observability/logs/forward-logs.md

```diff
@@ -125,3 +125,64 @@ To include a property, add it as a flag, for example `--protocol tcp`.
 This should let you connect to any service that has syslog endpoints.
 
 To start forwarding logs, once you've added the service [trigger a redeploy](../../development/troubleshoot.md#force-a-redeploy).
+
+## Log levels
+
+Though your application may output logs with distinct levels, as all logs are read from stdout, this distinction is lost and everything gets logged at the `INFO` level.
+
+To preserve the original log level you should use the language specific syslog module/package for logging.
+
+The following example code snippets show how logs can be written to syslog.
+
+{{< codetabs >}}
++++
+title=PHP
++++
+
+```php
+openlog("", LOG_PID, LOG_LOCAL1);
+syslog(LOG_LOCAL1|LOG_ERR, "Operation failed");
+closelog();
+```
+
+<--->
++++
+title=Python
++++
+
+```python
```

Comment:
I'd also document the standard logging support for syslog

```
import logging
import logging.handlers

logger = logging.getLogger(__name__)
logger.setLevel(logging.DEBUG)
handler = logging.handlers.SysLogHandler(address="/dev/log")
logger.addHandler(handler)
```

Reply:
Hi @ricardokirkner, have you had a chance to look at the latest changes made by @sivachandran, and if so are you happy with them?

Reply:
LGTM